### ISSUE TITLE
"Multiple Series Error" when changing visualization results in "empty" Queries page

### ISSUE THREAD

**Issue by volter** [grafana/grafana#17778]
**What happened**:

I changed the visualization type for a panel with two queries. This results in a "Multiple Series Error", which is OK. I then clicked the Queries symbol to remove or disable the second query. I end up on an "empty" page that only shows the left menu though.

**What you expected to happen**:

I expect the Queries page to load as normal.

**Anything else we need to know?**:

The funny thing is, if I start over and disable the second query first, swap Bar Gauge for Single stat and then enable second query again, I can navigate between the two pages just fine. Both are giving me the "Multiple Series Error", of course! 

**Environment**:
- Grafana version: 6.2.5
- Data source type & version: Elasticsearch
- User Browser: Firefox 67 and Chromium 73

**Comment by torkelo** [comment]
Can you open chrome dev tools and show us the error in the console? Sounds like there was a fatal js error .

**Comment by volter** [comment]
```
Error: "Multiple Series Error"
    setValues module.ts:392
    onDataReceived module.ts:272
    emit index.js:129
    emit actions.ts:29
    next QueriesTab.tsx:222
    subscribe PanelQueryRunner.ts:218
    componentDidMount VizPickerSearch.tsx:15
    React 3
        Ha
        Wa
        Os
    unstable_runWithPriority scheduler.production.min.js:18
    React 5
        Os
        Ls
        As
        Ps
        Cn
react-dom.production.min.js:198:186
    React 7
        fa
        callback
        ia
        ra
        Ha
        Wa
        Os
    unstable_runWithPriority scheduler.production.min.js:18
    React 5
        Os
        Ls
        As
        Ps
        Cn
Error: module.ts:392:12
    setValues module.ts:392
    onDataReceived module.ts:272
    <anonym> self-hosted:991
    emit index.js:129
    emit actions.ts:29
    next QueriesTab.tsx:222
    subscribe PanelQueryRunner.ts:218
    componentDidMount VizPickerSearch.tsx:15
    React 3
        Ha
        Wa
        Os
    unstable_runWithPriority scheduler.production.min.js:18
    React 5
        Os
        Ls
        As
        Ps
        Cn
    Cn self-hosted:1035
```

**Comment by marefr** [comment]
@volter is this still a problem? Could you try upgrading to 6.3.2 to verify if it's still a problem? Thanks

**Comment by volter** [comment]
Yes, still present in 6.3.2, but slightly different, I think. I'm a bit busy, but I'll get you the details soon.

**Comment by torkelo** [comment]
This should be fixed in 6.3.5
